lib/elixir/lib/kernel.ex

```diff
@@ -4020,24 +4020,18 @@ defmodule Kernel do
       is allowed and its results will be evaluated at runtime. Usually
       evaluates to the name of a module.
 
+      If the expression evaluates to a list, it will emit a warning,
+      since lists of expressions have been deprecated.
+
```

Comment:
We should remove this paragraph and improve the previous paragraph then. We should say: "the module to dispatch to". I don't think we need to say any expression is allowed (I would say that's the case for 99.9% of elixir macros).